Contents/Code/plex_api_helper.py

```diff
@@ -87,6 +87,30 @@ def setup_plexapi():
     return plex
 
 
+def is_field_locked(item, field_name):
+    # type: (any, str) -> bool
+    """
+    Check if the specified field is locked -- i.e. if the user has manually set a value for it and does not want
+    it to be updated by an automated process.
+
+    Parameters
+    ----------
+    item : any
+        The Plex item to check.
+    field_name : str
+        The name of the field to check.
+
+    Returns
+    -------
+    bool
+        True if the field is locked, False otherwise.
+    """
```

Comment:
```suggestion
    Returns
    -------
    bool
        True if the field is locked, False otherwise.
    """

    Examples
    --------
    # add an example please
```